Formula/pnpm.rb

```diff
@@ -20,14 +20,79 @@ class Pnpm < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "e0d669c48801ee9a3f0a49d99f81f2a50dcd1e879526b77a1367c1ae6daa9811"
   end
 
-  depends_on "node"
+  depends_on "node" => :build
```

Comment:
How can the tool written in node, run without node?

Reply:
It is bundled with https://github.com/vercel/pkg

Reply:
This formula creates a binary that combines the Node.js binary and the pnpm codes and pnpm can run without a system node.
This is like Deno's standalone binary.
This way is @zkochan wanted.